contracts/listing/listing-manager/IListingManager.sol

```diff
@@ -120,13 +127,22 @@ interface IListingManager is IContractEntity {
     function addLock(uint256 listingId, uint32 unlockTimestamp) external;
 
     /**
-     * @dev Marks the assets as being delisted. This operation in irreversible.
+     * @dev Marks the assets as being delisted.
      * After delisting, the asset can only be withdrawn when it has no active rentals.
```

Comment:
change this to After disabling listing, the asset should be enabled if lister wants it to be available on the market